src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/HmppsDomainEventEmitter.kt

```diff
@@ -92,10 +92,11 @@ class HmppsDomainEventEmitter(
     offenderNo: String,
     reason: PrisonerReceiveReason,
     prisonId: String,
+    occurredAt: Instant? = null,
   ) {
     PrisonerReceivedDomainEvent(
       PrisonerReceivedEvent(offenderNo, reason, prisonId),
-      Instant.now(clock),
+      occurredAt ?: Instant.now(clock),
       diffProperties.host,
```

Comment:
works better here than as a parameter default which would break any tests that mock this